components/layout/fragment.rs

```diff
@@ -97,6 +97,9 @@ pub struct Fragment {
     /// content edge of the fragment.
     pub border_padding: LogicalMargin<Au>,
 
+    /// The padding of the content box.
+    pub padding: LogicalMargin<Au>,
```

Comment:
Please don't add this field; it will hurt performance and memory usage. See the comment above. Instead, just use `border_padding - border_width()`.